sentry.client.config.ts

```diff
@@ -1,37 +1,33 @@
-import { init, Replay } from '@sentry/nextjs';
+import { SENTRY_DSN, SENTRY_ENABLE } from '@/next.constants.mjs';
 
-import { SENTRY_DSN, VERCEL_ENV } from '@/next.constants.mjs';
+// This lazy-loads Sentry on the Browser
+import('@sentry/nextjs').then(({ init }) =>
+  init({
+    // Only run Sentry on Vercel Environment
+    enabled: SENTRY_ENABLE,
+    // Tell Sentry where to send events
+    dsn: SENTRY_DSN,
+    // Disable Sentry Tracing as we don't need to have it
+    // as Vercel already does Web Vitals and Performance Measurement on Client-Side
+    enableTracing: false,
+    // We only want to capture errors from _next folder on production
+    // We don't want to capture errors from preview branches here
+    allowUrls: ['https://nodejs.org/_next'],
+    // Filter-out Events/Errors that are invalid for us
+    beforeSend: (event, hint) => {
+      const originalException = hint.originalException as Error;
 
-init({
-  // Only run Sentry on Vercel Environment
-  enabled: !!VERCEL_ENV,
-  // Tell Sentry where to send events
-  dsn: SENTRY_DSN,
-  // Percentage of events to send to Sentry (1% of them) (for performance metrics)
-  tracesSampleRate: 0.01,
-  // Percentage of sessions to sample (1%)
-  replaysSessionSampleRate: 0.01,
-  // Percentage of errors to sample (all of them)
-  replaysOnErrorSampleRate: 1.0,
-  // Support replaying client sessions to capture unhappy paths
-  integrations: [new Replay()],
-  // We only want to capture errors from _next folder on production
-  // We don't want to capture errors from preview branches here
-  allowUrls: ['https://nodejs.org/_next'],
-  // Filter-out Events/Errors that are invalid for us
-  beforeSend: (event, hint) => {
-    const originalException = hint.originalException as Error;
-
-    // All the Events we send must have a message and stack trace
-    if (originalException?.message && originalException?.stack) {
-      // All our Events come eventually from code that originates on node_modules
-      // so everything else should be discarded
-      // Even React Errors or other errors will eventually have node_modules in the code
-      if (String(originalException.stack).includes('node_modules')) {
-        return event;
+      // All the Events we send must have a message and stack trace
+      if (originalException?.message && originalException?.stack) {
+        // All our Events come eventually from code that originates on node_modules
+        // so everything else should be discarded
+        // Even React Errors or other errors will eventually have node_modules in the code
+        if (String(originalException.stack).includes('node_modules')) {
```

Comment:
Just a word of caution - this might not always hold true. For example, if your code throws inside a timeout or async handler, the stack leading up to the call might get lost.

Reply:
Hmmm, that is true. So far all the Exceptions we've seen, even unhandled promises inside Hooks, have had a stack trace that includes `node_modules`...

But this is such a good point. Do you believe it's best we remove this? (More false positives but also more actual exceptions?)

Reply:
Tough to say without knowing the codebase. I would probably remove the node_modules check first and re-evaluate after this PR is deployed to see if you need it.